src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -48,13 +48,14 @@ static bool DetectCPUFeatures();
 
 extern RhConfig * g_pRhConfig;
 
-#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64)
+#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64) || defined(HOST_LOONGARCH64)
 // This field is inspected from the generated code to determine what intrinsics are available.
 EXTERN_C int g_cpuFeatures;
 int g_cpuFeatures = 0;
 
 // This field is defined in the generated code and sets the ISA expectations.
 EXTERN_C int g_requiredCpuFeatures;
+int g_requiredCpuFeatures = 0;
```

Comment:
This should be emitted by the AOT compiler. Why is it not happening for LoongArch?

This addition is the source of the build breaks that you are seeing.

If you do not have any intrinsics support yet, maybe all changes in this file should be reverted?

Reply:
Thank you for helping me discover this mistake, I missed the change in src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ExpectedIsaFeaturesRootProvider.cs.
Now it may seem right.
Thanks!